blaze/expr/broadcast.py

```diff
@@ -65,8 +65,14 @@ def _name(self):
 
     @property
     def _full_expr(self):
-        return self._scalar_expr._subs(dict(zip(self._scalars,
-                                                self._children)))
+        return self._scalar_expr._subs(
+            dict(zip(self._scalars, self._children))
+        )
+
+    @property
+    def _args(self):
+        # XXX: is this a hack?
```

Comment:
Ah, okay. Lets leave the comment for now then and maybe revisit the path algorithm some time in the future.